### PR TITLE
S158685 Added skip parameter documentation

### PR DESCRIPTION
**Pull request by bolst03**

### CODE REVIEW

File: docs/messages.md [hunk 1]
```diff
@@ -156,6 +156,7 @@ Lists messages from a flow, filtered by parameters.
 | event | Filter messages by event type. Leave blank to get all messages, which is also the default. Multiple event types can be specified by separating them with commas. |
 | limit | Maximum number of messages to return. Accepted values range from 1 to 100. Defaults to 30. |
 | sort | Descending order is used by default. Use parameter value `asc` for ascending order. Note: the response JSON will always have the matching messages in ascending order by their `id`. The sort parameter is for toggling between fetching the latest or the oldest matching messages. |
+| skip | Skip a number of messages in the response. Use for paging results. |
```
Comment (srmelody-rally) [inline docs/messages.md]: do you want to mention this is only support with search?  And perhaps mention since_id is not used for search (only the main message api?)

Comment (bolst03) [reply docs/messages.md]: yes

**Comment by bolst03** [comment]
@srmelody-rally updated

**Review by srmelody-rally — COMMENTED** [review]
:shipit:

**Pull request merged**